docs/hw/mechanical.md

```diff
@@ -10,13 +12,16 @@ IMPORTANT: Always remove the Create® 3 robot’s battery prior to dismantling,
 ![The Create® 3 Top Drawing](data/top_drawing.svg "Create® 3 Top Drawing")
 ### Side View
 ![The Create® 3 Side Drawing](data/side_drawing.svg "Create® 3 Side Drawing")
+<sub>Wheel travel is approximately 30 mm.</sub>
 ### Bottom View
 ![The Create® 3 Bottom Drawing](data/bottom_drawing.svg "Create® 3 Bottom Drawing")
 ### Cargo Bay - Top
 ![The Create® 3 Cargo Bay Drawing](data/cargo_bay_drawing.svg "Create® 3 Cargo Bay Drawing")
 
 ## Downloadable 3D Model
-Download this [STEP model (17.7 MB)](data/iRobot_Create_3_Public_Model.step) to help design attachments or payloads for Create® 3.
+![Create® 3 Model](data/3d_model.jpg "3D Model of Create® 3")
+Download this [STEP model (17.7 MB)](data/iRobot_Create_3_Public_Model.step)[^1] to help design attachments or payloads for Create® 3.
+[^1]: Copyright © 2021-22 iRobot Corporation. ​​All Rights Reserved.
```

Comment:
nice!
this renders much more nicely than the footers notes that we have in the electrical and hookup examples pages

Reply:
I agree, but not sure how to effect this on those other pages, since there's not a particular thing that is being footnoted.

Reply:
Maybe we can assign the footnote to the first mention of "usb-c", "Bluetooth", "RaspberryPi", etc

Reply:
Yeah, I think this works pretty well actually. I made the change in commit b1041db if you want to check it out.